app/Http/Controllers/ReportsController.php

```diff
@@ -1152,22 +1154,14 @@ public function sentAssetAcceptanceReminder(Request $request) : RedirectResponse
         }
 
         // Only send notification if assigned
-        if ($assetItem->assignedTo) {
+        if ($assetItem->assignedTo?->email) {
+                Mail::to($assetItem->assignedTo->email)->send((new CheckoutAssetMail($assetItem, $assetItem->assignedTo, $logItem->user, $logItem->note, $acceptance))->locale($assetItem->assignedTo?->locale));
 
-            if (!$assetItem->assignedTo->locale) {
-                Notification::locale(Setting::getSettings()->locale)->send(
-                    $assetItem->assignedTo,
-                    new CheckoutAssetNotification($assetItem, $assetItem->assignedTo, $logItem->user, $acceptance, $logItem->note)
-                );
             } else {
-                Notification::send(
-                    $assetItem->assignedTo,
-                    new CheckoutAssetNotification($assetItem, $assetItem->assignedTo, $logItem->user, $acceptance, $logItem->note)
-                );
+                Mail::to($assetItem->assignedTo->email)->send((new CheckoutAssetMail($assetItem, $assetItem->assignedTo, $logItem->user, $logItem->note, $acceptance)));
```

Comment:
I think the `if` check for `$assetItem->assignedTo->locale` was accidentally removed?

This line uses `$assetItem->assignedTo->email` in an `else` that checked to see if it existed. In other words, this `else` could fail because `$assetItem->assignedTo->email` is falsy here.

Reply:
updated, also made it a bit more readable.